commands.json

```diff
@@ -982,7 +1037,8 @@
           {
             "name": "highlight",
             "type": "pure-token",
-            "token": "HIGHLIGHT"
+            "token": "HIGHLIGHT",
+            "summary": "Highlights terms in the search results, with customizable tags for emphasis."
```

Comment:
Should we mention it is supported only for HASH?
[Limitation](https://redis.io/docs/latest/develop/interact/search-and-query/indexing/#index-limitations)
> No HIGHLIGHT and SUMMARIZE support

Reply:
Wrote that it isn't supported for json documents.